### PR TITLE
feat: implement ite6 guesser

### PR DESCRIPTION
**Pull request by cpendery**
Implements a guesser for `ite6` and `slsa` documents

Closes https://github.com/guacsec/guac/issues/34
Closes https://github.com/guacsec/guac/issues/35

### CODE REVIEW

File: pkg/handler/processor/guesser/format_ite6.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+package guesser
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/guacsec/guac/pkg/handler/processor"
+)
+
+type attestation struct {
```
Comment (lumjjb) [inline pkg/handler/processor/guesser/format_ite6.go:25]: hmm, maybe use the intoto library here for the statement, like https://github.com/guacsec/guac/pull/39/files#diff-57f9f62d5b3797cdf1646065bad24ebaeed1d19eeb717a918f7b7e0f60623e5fR82

**Review by lumjjb — CHANGES_REQUESTED** [review]
Thanks for opening the PR @cpendery ! Have a couple comments for changes - thanks!

File: pkg/handler/processor/processor.go [hunk 2]
```diff
@@ -64,6 +64,8 @@ type FormatType string
 // Format* is the enumerables of FormatType
 const (
 	FormatJSON    FormatType = "JSON"
+	FormatSLSA    FormatType = "SLSA"
+	FormatITE6    FormatType = "ITE6"
```
[the remark this answers is not in the document]
Comment (mihaimaruseac) [reply pkg/handler/processor/processor.go:68]: +1. `FormatType` is the format on the wire, `DocumentType` is the type of the document, how it should be interpreted.

**Review by mihaimaruseac — APPROVED** [review]

**Comment by cpendery** [comment]
I signed the cla @lumjjb @mihaimaruseac if you can rerun that job for me

**Review by pxp928 — APPROVED** [review]

**Review by lumjjb — APPROVED** [review]
LGTM! Thanks @cpendery !!

**Pull request merged**